qlog/frame.go

```diff
@@ -0,0 +1,360 @@
+package qlog
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/lucas-clemente/quic-go/internal/protocol"
+	"github.com/lucas-clemente/quic-go/internal/wire"
+)
+
+type frame struct {
+	Frame interface{}
+}
+
+type streamType protocol.StreamType
+
+func (s streamType) String() string {
+	switch protocol.StreamType(s) {
+	case protocol.StreamTypeUni:
+		return "unidirectional"
+	case protocol.StreamTypeBidi:
+		return "bidirectional"
+	default:
+		panic("unknown stream type")
+	}
+}
+
+func escapeStr(str string) []byte { return []byte("\"" + str + "\"") }
```

Comment:
This will break if the string has quotation marks?

Reply:
Yes, and I removed this function in the commit that switched from standard library JSON to GoJay.